### PR TITLE
feat: Render contentful preview API results when preview query param …

### PR DESCRIPTION
**Pull request by eddieferrer**
…is active for ContentfulPage component

* Uses cookies to create an apollo link that automatically adds the preview variable to the apollo query when necessary
VUE-671

Uses a cookie as discussed in https://github.com/kiva/ui/pull/3214 this allows the components, for example: `src/pages/ContentfulPage.vue` to be totally agnostic to the preview query param on the route

### CODE REVIEW

File: src/api/ContentfulPreviewLink.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { ApolloLink } from 'apollo-link';
+
+export default ({ cookieStore }) => {
+	return new ApolloLink((operation, forward) => {
+		// Only do this on contentful queries
+		if (operation.operationName.includes('contentful')) {
+			// Fetch current basket id from cookie
```
Comment (emuvente) [inline src/api/ContentfulPreviewLink.js:7]: basket id?

**Review by emuvente — APPROVED** [review]
👍

**Review by mcstover — APPROVED** [review]
Very cool!

File: src/api/ContentfulPreviewLink.js (hunk 1 above)
Comment (eddieferrer) [reply src/api/ContentfulPreviewLink.js:7]: > basket id?

... Is the  apollo link I copied and pasted 😆

**Pull request merged**